### PR TITLE
metrics.md updated

### PR DESCRIPTION
**Pull request by Aparnarr**
Signed-off-by: Aparna <Aparna.Rr@emc.com>

1. Formatted Heading
 2. Minor editing

### CODE REVIEW

File: metrics.md [hunk 1]
```diff
@@ -8,39 +8,36 @@ You may obtain a copy of the License at
...
-## Metrics
+# Metrics
 
-Pravega metrics are collected and exposed to Flink metrics framework when using `FlinkPravegaReader` or `FlinkPravegaWriter` instances to perform read and write operations on Pravega streams.
+Pravega metrics are collected and exposed to Flink metrics framework when using `FlinkPravegaReader` or `FlinkPravegaWriter` instances to perform read and write operations on Pravega Streams.
 
-### Reader Metrics
+## Reader Metrics
 
-The following metrics are exposed and made available as Flink metrics for `FlinkPravegaReader` related operations.
```
Comment (Aparnarr) [inline metrics.md:17]: looks repeated so i removed?!

File: metrics.md [hunk 2]
```diff
@@ -8,39 +8,36 @@ You may obtain a copy of the License at
     http://www.apache.org/licenses/LICENSE-2.0
 -->
 
-## Metrics
+# Metrics
 
-Pravega metrics are collected and exposed to Flink metrics framework when using `FlinkPravegaReader` or `FlinkPravegaWriter` instances to perform read and write operations on Pravega streams.
```
Comment (Aparnarr) [inline metrics.md:13]: any links to Flinkpravegareader and writer??

Comment (vijikarthi) [reply metrics.md:13]: Yes, please add the link to the streaming.md (FlinkPravegaReader & FlinkPravegaWriter) sections

File: metrics.md [hunk 3]
```diff
@@ -8,39 +8,36 @@ You may obtain a copy of the License at
     http://www.apache.org/licenses/LICENSE-2.0
 -->
 
-## Metrics
+# Metrics
 
-Pravega metrics are collected and exposed to Flink metrics framework when using `FlinkPravegaReader` or `FlinkPravegaWriter` instances to perform read and write operations on Pravega streams.
+Pravega metrics are collected and exposed to Flink metrics framework when using `FlinkPravegaReader` or `FlinkPravegaWriter` instances to perform read and write operations on Pravega Streams.
```
Comment (vijikarthi) [inline metrics.md]: Can we change it to 
> Pravega metrics are collected and exposed via Flink metrics while performing read and write operations on Pravega stream using `FlinkPravegaReader` or `FlinkPravegaWriter`.

File: metrics.md [hunk 4]
```diff
@@ -8,39 +8,36 @@ You may obtain a copy of the License at
...
+The following metrics are exposed and made available as Flink metrics for `FlinkPravegaReader` related operations:
 
 Metric Name                |Description|
 |-----------------|-----------------------------------------------------------------------|
-|`readerGroupName`|The name of the reader group.|
-|`scope`|The scope name of the reader group.|
-|`streams`|The streams that are part of the reader group. The stream name will be a fully qualified name i.e., `scope/stream`|
+|`readerGroupName`|The name of the ReaderGroup.|
+|`scope`|The scope name of the ReaderGroup.|
+|`streams`|The streams that are part of the ReaderGroup. The stream name will be a fully qualified name i.e., `scope/stream`|
```
Comment (vijikarthi) [inline metrics.md]: |`streams`|The streams that are part of the `ReaderGroup` will be listed as comma separated values. The stream name will be a fully qualified name i.e., `scope/stream`|

File: metrics.md [hunk 5]
```diff
@@ -8,39 +8,36 @@ You may obtain a copy of the License at
...
 
-### Writer Metrics
+## Writer Metrics
+
+For `FlinkPravegaWriter` related operations, only the stream name is exposed:
 
-For `FlinkPravegaWriter` related operations, only the stream name is exposed.
+Metric Name                 |Description|
+|-----------------|-----------------------------------------------------------------------|
+|`streams`  |The streams that are part of the ReaderGroup. The stream name will be a fully qualified name i.e., `scope/stream`|
```
Comment (vijikarthi) [inline metrics.md]: The column width for the "Metric Name" is less. Please use the same column width from the pervious section.

File: metrics.md [hunk 6]
```diff
@@ -8,39 +8,36 @@ You may obtain a copy of the License at
...
 
-### Querying Metrics
+## Querying Metrics
 
-The metrics can be viewed either from Flink UI or using the Flink Rest API (see below for example).
+The metrics can be viewed either from Flink UI or using the Flink `REST` API, see below:
 
-```
+```java
```
Comment (vijikarthi) [inline metrics.md:41]: In this section, can you please replace all the occurrences of `SOURCE-READER-NAME` to `SOURCE-OPERATOR-NAME`

**Review by vijikarthi — CHANGES_REQUESTED** [review]

File: metrics.md (hunk 2 above)
Comment (Aparnarr) [reply metrics.md:13]: done

File: metrics.md (hunk 3 above)
Comment (Aparnarr) [reply metrics.md]: Accepted

File: metrics.md (hunk 4 above)
Comment (Aparnarr) [reply metrics.md]: Accepted suggestion

File: metrics.md (hunk 6 above)
Comment (Aparnarr) [reply metrics.md:41]: done

File: metrics.md (hunk 5 above)
Comment (Aparnarr) [reply metrics.md]: markdown columns width is determined by the longest cell in the column,

**Review by vijikarthi — APPROVED** [review]

**Pull request merged**